client/www/scripts/modules/manager/manager.controllers.js

```diff
@@ -638,20 +682,21 @@ Manager.controller('ManagerMainController', [
 
     function updateHostEnv(host, env) {
       var deferred = $q.defer();
-      var command =  {
-        cmd: 'env-set',
-        env: env
-      };
 
-      if (host) {
-        host.action(command, function(err, res) {
-          if (err) {
-            deferred.reject(err);
+      getPmService(host).then(function(service) {
+        // TODO: this should be changed to use setEnvs when strong-mesh-models
+        //       is updated to v6
+        //       service.setEnvs(env);
```

Comment:
strongloop/strong-mesh-models#52 has been merged, which back-ports `setEnvs` to 5.x :-)